src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -19,6 +19,38 @@
 
 #include <unordered_set>
 
+namespace {
+using namespace chatterino;
+MessagePtr generateBannedMessage(bool confirmedBan)
+{
+    const auto linkColor = MessageColor(MessageColor::Link);
+    const auto accountsLink = Link(Link::Reconnect, QString());
+    const auto curUser = getApp()->accounts->twitch.getCurrent();
+    const auto bannedText = confirmedBan
+                                ? QString("You were banned from this channel!")
+                                : QString("You were forced off this channel.");
```

Comment:
I feel like this message on its own does not provide users with enough information about what happened. (Was does "forced off" mean?)

I'm not sure what a better message would be but I think even something like `Your connection to this channel was dropped.` would be preferable. Open to discussion though

Reply:
The message is meant to convey the user's account being forcibly `PART`ed from the channel, *forced off the channel*.

Reply:
Yes, that makes sense to me and people familiar with TMI. I just think it could lead to confusion for the average user.
Maybe `Your connection to this channel was dropped by Twitch.` or similar could be a viable compromise?